### PR TITLE
(PC-41945)[API] test: add sandbox data for artist page QA edge cases

### PR DESCRIPTION
**Pull request by tntatla-pass**
[ticket associé](https://passculture.atlassian.net/browse/PC-41945)

Ajout de deux nouvelles fonctions de peuplement sandbox pour couvrir les cas edge de la page artiste :

- La fonction _create_multi_category_artist : crée Charles Aznavour avec des offres dans 4 sous-catégories (CD, Vinyle, Cinéma, Livre)
- La fonciton_create_artist_without_offers : crée un artiste sans aucune offre liée

Les deux fonctions sont couvertes par des tests unitaires (test_create_artists.py).

### CODE REVIEW

File: api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py [hunk 1]
```diff
@@ -566,6 +568,50 @@ def _create_library_with_writers() -> None:
...
+    """
+    artist_factories.ArtistFactory.create(
+        name="Artiste sans offre",
+        description="cas limite QA — artiste sans aucune offre liée",
+        biography=(
+            "Cet artiste est volontairement dépourvu d'offres afin de valider que la page artiste"
+            " n'affiche aucune playlist de catégorie lorsque aucune œuvre n'est rattachée."
+        ),
+        wikidata_id="Q00000001",
+        wikipedia_url=None,
```
Comment (prouzet-pass) [inline api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py]: Détail, simple question : est-il pertinent/réaliste d'associer un `wikipedia_id` sans `wikipedia_url` ?

**Review by prouzet-pass — APPROVED** [review]

File: api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py (hunk 1 above)
Comment (tntatla-pass) [reply api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py]: Hmmm je pense que le wikidata_id n'est pas pertinent pour ce cas de test, çà devrait passer sans normalement
